runners-manager/vm_creation/openstack.py

```diff
@@ -25,12 +26,36 @@
 neutron = neutronclient.v2_0.client.Client(session=session, region_name=region)
 
 
-def script_init_runner(name, token, labels, group):
-    installer = link_download_runner('scalanga-devl')
-    return f"""#!/bin/bash
+def install_docker(image: str):
+    if 'centos' in image.lower():
+        return """
 sudo yum install -y bind-utils yum-utils
 sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
 sudo yum install -y epel-release docker-ce docker-ce-cli containerd.io
+""" # noqa
```

Comment:
Coment for all the scripts:
* Create separate file for all of them that you open and use
* If needed you can add jinja template